.github/workflows/try-in-web-ide.yaml

```diff
@@ -18,16 +18,16 @@ on:
 
 jobs:
   add-link:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
     steps:
       - name: Add DevSandbox link
-        uses: redhat-actions/try-in-web-ide@v1
+        uses: redhat-actions/try-in-web-ide@v1.2
         with:
           github_token: ${{ secrets.GITHUB_TOKEN }}
           add_comment: true
           add_status: false
       - name: Add Dogfooding link
-        uses: redhat-actions/try-in-web-ide@v1
+        uses: redhat-actions/try-in-web-ide@v1.2
```

Comment:
why 1.2 here? are we avoiding 1.3+ ?

Reply:
Currently, 1.2 is the latest - https://github.com/redhat-actions/try-in-web-ide/releases

Reply:
Currently, 1.2 is the latest - https://github.com/redhat-actions/try-in-web-ide/releases

Reply:
yes, but @1 == @1.x so this change is actually PINNING to 1.2 rather than using 1.latest